src/screens/TopicPageScreen/MemoizedListComponent.js

```diff
@@ -116,6 +120,35 @@ const RenderListFeed = (props) => {
     initialSetup(item);
     checkVotesHandle();
   }, [item]);
+
+  const hasComment = getCommentLength(item.latest_reactions.comment) > 0;
+
+  const isBlurred = item?.isBlurredPost && item?.anonimity;
+
+  const commentHeight = () => {
+    if (isBlurred && !hasComment) {
+      return 0;
+    }
+    const isSingleComment = getTotalReaction(item) === 1;
+    const commentSectionHeight = getHeightReaction() - getHeightFooter();
+    return isSingleComment ? commentSectionHeight - 20 : commentSectionHeight;
+  };
+
+  const {onLayoutTopicChip} = useCalculationContent();
+  const calculateLineTopicChip = (nativeEvent) => {
+    onLayoutTopicChip(nativeEvent, 1);
+  };
+
+  const topicBottomPosition = () => {
+    if (hasComment) {
+      if (getTotalReaction(item) === 1) {
+        return getHeightReaction() - 20;
+      }
+      return getHeightReaction();
+    }
+    return getHeightFooter();
+  };
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

New logic has been introduced to calculate whether a post has comments, whether it's blurred, and the height of the comment section. This seems fine, but ensure that all the functions and variables used here (`getCommentLength`, `getTotalReaction`, `getHeightReaction`, `getHeightFooter`) are defined and work as expected.
```javascript
const hasComment = getCommentLength(item.latest_reactions.comment) > 0;
const isBlurred = item?.isBlurredPost && item?.anonimity;
const commentHeight = () => {
  if (isBlurred && !hasComment) {
    return 0;
  }
  const isSingleComment = getTotalReaction(item) === 1;
  const commentSectionHeight = getHeightReaction() - getHeightFooter();
  return isSingleComment ? commentSectionHeight - 20 : commentSectionHeight;
};
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->